StudentsGroup.java

```diff
@@ -0,0 +1,66 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class StudentsGroup {
+    private Student leader;
+    private List<Student> students;
+    private List<String> tasks;
+    private Map<Student, Boolean> tasksStatus;
```

Comment:
Structure to define compleated tasks is wrong. How do you decide which task is compleated. Use `Map<String, List<Student>> compleatedTasks = new HashMap<>();` instead